pkg/git/gitlab/gitlab.go

```diff
@@ -230,23 +230,27 @@ func (g *Gitlab) FullURL() *url.URL {
 // it will return false.
 func (g *Gitlab) IsType(URL *url.URL) (bool, error) {
 
+	if strings.Contains(URL.Host, "gitlab.com") {
```

Comment:
I really don't like using such "magic strings".
We could use the [/version](https://docs.gitlab.com/ee/api/version.html) endpoint?

Reply:
It's not available on gitlab.com:
```
➜ curl https://gitlab.com/version
<html><body>You are being <a href="https://gitlab.com/users/sign_in">redirected</a>.</body></html>%    
```

Reply:
Another idea would be to just return `true` and think about this problem again once we actually implement other Git providers.

I'm not even sure anymore it makes sense to auto discover the git provider like this. We might as well just include another field in the credentials secret to state which git provider it is. This just seems very brittle and probably causes more errors and confusion than it helps.

What do you think?

Reply:
Fine by me.

Until we implement the change correclty I'll do the `return true` thing.